index.html

```diff
@@ -63,7 +63,7 @@ <h3 class="Title_MyRecentWorks">My recent Works</h3>
       </div>
       <div class="cnt_info_main_card">
         <h3 class="MainCardTitle Title2_dark">Multi-Post Stories</h3>
-        <p class="MainCardDescrip Simpletext">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <p class="MainCardDescrip">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
```

Comment:
- [ Required ] As requested by a previous reviewer I think it would be better to change the `font-family` of all project descriptions to `Inter` to match the design provided on the Figma template. Currently, the `font-family` in your design is `Poppins` while in the Figma design, the project descriptions font is `Inter`. You can achieve this by using the code snippet below.

```CSS
.CardDescrip {
  color: var(--color-font-text1_card);
  font-family: var(--font-card-text);
  font-size: 1rem;
}
```

Reply:
OMG. I changed the font-family at the first card, and forgot the others. Excuses.
Its Done. Thank you.